src/helpers/getStations.ts

```diff
@@ -0,0 +1,32 @@
+import { Promise } from 'es6-promise';
+import bus_station  from '../models/bus_station';
+import db from '../db'
+db();
+
+const googleMapsClient = require('@google/maps').createClient({
```

Comment:
I think that it's not the first use of creatingClient.. shouldn't we take that to another module ?